Gemfile.lock

```diff
@@ -0,0 +1,24 @@
+GEM
```

Comment:
Nit: I don't think you need to check this file in (see https://github.com/killbill/killbill-stripe-demo/blob/master/.gitignore#L2).